### ISSUE TITLE
raining loss is not decreasing when using the Roberta pre-trained model from the transformers library

### ISSUE THREAD

**Issue by ZahraAbbasiantaeb** [huggingface/transformers#8173]
I load the Roberta pre-trained model from the transformers library and use it for the sentence-pair classification task. The loss function used to decrease during the training per epoch until the last week, but now even though all of the parameters, including the batch size and the learning rate have the same value, when I fit my model the value of the loss function is not decreasing. I am a little bit confused and I have trained my model using various parameters and also I utilized another code in PyTorch, but still, the loss function is not decreasing. Can anyone help me to figure out the problem?

here is the link to my code:
https://colab.research.google.com/drive/1CFg41KDHJSJNkehJOHbp3gfXRdva60oW?usp=sharing

and the dataset:
https://drive.google.com/drive/folders/1CUH_z_HI31-yfj8hOmRfJBKRKe_BNkku

**Comment by LysandreJik** [comment]
Hello! Could you open a post on the [forum](https://discuss.huggingface.co) instead? We try to keep issues for bugs only.
